core/src/services/gcs/writer.rs

```diff
@@ -27,6 +27,8 @@ use crate::ops::OpWrite;
 use crate::raw::*;
 use crate::*;
 
+/// It's recommended that you use at least 8 MiB for the chunk size.
+const DEFAULT_GCS_MINIMUM_PART_SIZE: usize = 8 * 1024 * 1024;
```

Comment:
GCS requires size aligned with 256KiB, so `MINIMUM` is incorrect. And gcs doesn't have concept of `PART_SIZE`.

How about use `DEFAULT_WRITE_FIXED_SIZE` instead?